rust/otel-arrow-rust/.gitignore

```diff
@@ -9,3 +9,6 @@ Cargo.lock
 
 # These are backup files generated by rustfmt
 **/*.rs.bk
+
+# This is a library, do not check-in locked dependencies.
+Cargo.lock
```

Comment:
There is already an entry for `Cargo.lock` at line 8.

https://github.com/open-telemetry/otel-arrow/blob/233ad8a5351e0f10a7d0b8d39205836131faf79f/rust/otel-arrow-rust/.gitignore#L6-L8